File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.java
import java.util.*;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        if(arr1.length == arr2.length){
            int arr1_sum = Arrays.stream(arr1).sum();
            int arr2_sum = Arrays.stream(arr2).sum();
            if(arr1_sum == arr2_sum) return 0;
            else if(arr1_sum > arr2_sum) return 1;
            else return -1;
        } else if(arr1.length > arr2.length) return 1;
        return -1;
    }
}

File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.cpp
#include <string>
#include <vector>
#include <numeric>
using namespace std;

int solution(vector<int> arr1, vector<int> arr2) {
    int answer = 0;
    if(arr1.size() == arr2.size()){
        int arr1_sum = accumulate(arr1.begin(),arr1.end(), 0);
        int arr2_sum = accumulate(arr2.begin(),arr2.end(), 0);
        if(arr1_sum == arr2_sum){
            return 0;
        }else if(arr1_sum > arr2_sum){
            return 1;
        }else{
            return -1;
        }
    } else if(arr1.size() < arr2.size()){
        return -1;
    }
    return 1;
}